radio/src/mixer.cpp

```diff
@@ -593,15 +590,22 @@ void evalFlightModeMixes(uint8_t mode, uint8_t tick10ms)
 #endif
 
 #if defined(HELI)
+#if defined(PCBTARANIS)
+  int heliEleValue = getValue(g_model.swashR.elevatorSource);
+  int heliAilValue = getValue(g_model.swashR.aileronSource);
+#else
+  int16_t heliEleValue = anas[ELE_STICK];
+  int16_t heliAilValue = anas[AIL_STICK];
+#endif
   if (g_model.swashR.value) {
-    uint32_t v = ((int32_t)HELI_ANAS_ARRAY(ELE_STICK)*HELI_ANAS_ARRAY(ELE_STICK) + (int32_t)HELI_ANAS_ARRAY(AIL_STICK)*HELI_ANAS_ARRAY(AIL_STICK));
+    uint32_t v = ((int32_t)heliEleValue*heliEleValue + (int32_t)heliAilValue*heliAilValue);
     uint32_t q = calc100toRESX(g_model.swashR.value);
     q *= q;
     if (v>q) {
       uint16_t d = isqrt32(v);
       int16_t tmp = calc100toRESX(g_model.swashR.value);
-      HELI_ANAS_ARRAY(ELE_STICK) = (int32_t) HELI_ANAS_ARRAY(ELE_STICK)*tmp/d;
```

Comment:
This is strange to me, we wrote anas (and heliAnas) as many times as active flight modes? I think it was a bug.